include/trgdataformats/TriggerCandidateData.hpp

```diff
@@ -53,7 +63,7 @@ struct TriggerCandidateData
   };
 
   // Update this version number if there are any changes to the in-memory representation of this class!
-  static constexpr version_t s_trigger_candidate_version = 1; // NOLINT(build/unsigned)
+  static constexpr version_t s_trigger_candidate_version = 2; // NOLINT(build/unsigned)
```

Comment:
Ah, that was one of the things we were supposed to discuss at today's trigger meeting.
On one hand we are adding many new types, on the other hand structure and size of a TC doesn't change.
I don't have a strong opinion here, so lgtm